app/controllers/cim_instance_controller.rb

```diff
@@ -147,7 +147,7 @@ def process_show(associations = {})
     else
       whitelisted_key = associations.keys.find { |key| key == @display }
       if whitelisted_key.present?
-        model_name = whitelisted_key.singularize.classify.constantize
+        model_name = whitelisted_key.singularize.classify
```

Comment:
While we're here: `classify` == `singularize.camelize`, so no need for the explicit `singularize` here.